lib/run.js

```diff
@@ -50,19 +53,19 @@ function run(config, done) {
                     total: tasks.length,
                     info: info,
                     testCmd: testCmd,
-                    installCmd: "npm i" + (config.registry ? (" --registry " + config.registry) : "")
+                    installCmd: (config.useYarn ? "yarn add" : "npm i") + (config.registry ? (" --registry " + config.registry) : "")
```

Comment:
So, yarn does not provide a way to set the registry? I think, we should then call `finish()` with an error that this is not supported with yarn. They need to set the `--force-npm` flag.